server/src/main/java/org/apache/iotdb/db/engine/StorageEngine.java

```diff
@@ -1083,6 +1087,14 @@ protected void getSeriesSchemas(InsertPlan insertPlan, VirtualStorageGroupProces
     }
   }
 
+  public DataRegion createDataRegion(DataRegionId regionId, String sg, long ttl) {
+    return null;
```

Comment:
null?  why?

Reply:
StorageEngine will implement this function in a later PR, and I've added comments and TODO